core/src/components/picker-column-internal/test/disabled/picker-column-internal.e2e.ts

```diff
@@ -159,3 +159,40 @@ configs({ modes: ['ios'], directions: ['ltr'] }).forEach(({ title, config }) =>
     });
   });
 });
+
+/**
+ * This behavior does not vary across directions.
+ */
+configs({ directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
+  test.describe(title('picker-column-internal: disabled column rendering'), () => {
+    test.beforeEach(async ({ page }) => {
+      await page.goto('/src/components/picker-column-internal/test/disabled', config);
+    });
+
+    test('disabled column should not have visual regressions', async ({ page }) => {
+      const disabledColumn = page.locator('#column-disabled');
+      await disabledColumn.evaluate((el: HTMLIonPickerColumnInternalElement) => {
+        el.value = 11;
+      });
+      await page.waitForChanges();
+
+      await expect(disabledColumn).toHaveScreenshot(screenshot('picker-internal-disabled-column'));
```

Comment:
We should dig into this test more. Playwright is already reporting it as flaky.

Reply:
It's still flaking. I can't reproduce locally 😕

Reply:
Can we try loading the test with the `value` already set? The problem might be that Playwright isn't waiting for the scroll to finish before taking a screenshot.

Reply:
Done in 46616677852bdaaa26439ff2b3183b970c805926 and I also made a change in 7a3771a220170a5ed91dce629d44bd41d281bef2 that hopefully will help.

Reply:
I meant that we should try loading the picker with `value="11"` already set. So you should be able to do `value="11"` here: https://github.com/ionic-team/ionic-framework/blob/46616677852bdaaa26439ff2b3183b970c805926/core/src/components/picker-column-internal/test/disabled/index.html#L56 and then remove this line: https://github.com/ionic-team/ionic-framework/blob/46616677852bdaaa26439ff2b3183b970c805926/core/src/components/picker-column-internal/test/disabled/index.html#L81

For https://github.com/ionic-team/ionic-framework/commit/7a3771a220170a5ed91dce629d44bd41d281bef2 this causes a regression where the selected item is no longer highlighted when the picker column is disabled. You can see this at `src/components/picker-column-internal/test/disabled` on MD mode. The "11" option in "Column disabled" is no longer highlighted in blue.

According to your comment in https://github.com/ionic-team/ionic-framework/pull/28511/files#diff-5e7b473017ccb3a8be0fa9c7c9d8e1fc393b64ddfe719d5f3274e212ba09b3fdR416 this item should be highlighted blue.

Reply:
Updated. Also, I added a screenshot for the disabled calendar wheel to capture that intended behavior.

Reply:
> I don't understand how setting the value directly changes anything for the scrolling if the items don't get set until the script? It can't scroll until the items are there any more than it can scroll if the value isn't there, right? 

The issue that I raised was the value was being set after the component loaded: https://github.com/ionic-team/ionic-framework/pull/28511/files/b7e42befeabbbb96044bbaa0aeb6ba6668baa547#diff-875e8dba2863da31c37c64b445f9c01418f2ee542625acf814f7a8cab7464b19R174

Your old test loaded the picker with `value="12"`. The picker spends time scrolling to value 12. The test then set the value to `11`, so the picker then needed to spend additional time scrolling to value 11. This additional value update can take a few hundred ms especially on CI.

By setting up the test with the correct configuration on load we a) reduce the run time of the test and b) reduce the likelihood of flaky tests.